spec/views/application_navbar_spec.rb

```diff
@@ -0,0 +1,14 @@
+require "rails_helper"
+
+describe "application/_navbar.html.erb" do
+  let(:current_user) { instance_double("user") }
```

Comment:
Prefer not to use `let` blocks but yay for starting on view specs :)

Reply:
lol, only because I was going down the bad path now!

Reply:
Done by @elle sha: 2985975